src/unity/python/turicreate/toolkits/one_shot_object_detector/one_shot_object_detector.py

```diff
@@ -11,15 +11,16 @@
 from turicreate.toolkits._model import PythonProxy as _PythonProxy
 from turicreate.toolkits.object_detector.object_detector import ObjectDetector as _ObjectDetector
 from turicreate.toolkits.one_shot_object_detector.util._augmentation import preview_augmented_images
+from turicreate.toolkits._internal_utils import _raise_error_if_column_exists
 
 def create(data,
            target,
            backgrounds=None,
            batch_size=0,
-           max_iterations=0,
+           max_iterations=1,
```

Comment:
I don't think the default should be 1 :)

Reply:
Will change it